src/Webroot.php

```diff
@@ -148,15 +154,19 @@ public function putassets($sspak) {
 		$assetsParentArg = escapeshellarg(dirname($assetsPath));
 
 		// Move existing assets to assets.old
-		$this->exec("if [ -d {$assetsPath} ]; then mv {$assetsPath} {$assetsOldPath}; fi");
+		if (file_exists($assetsPath)) {
```

Comment:
Why have we moved the `file_exists` check into PHP and away from bash, which would execute under the sudo user (which is probably important?)

Reply:
I will push a version without file_exists shortly. Just testing it now.